skins/10h16/js/lib/reducers/donation_input_validation.js

```diff
@@ -24,7 +24,23 @@ var _ = require( 'underscore' ),
 		confirmSepa: _.clone( defaultFields ),
 		confirmShortTerm: _.clone( defaultFields )
 	},
-
+	setValidityOfPaymentData = function ( state, action ) {
+		if ( action.type === 'SELECT_AMOUNT' ) {
+			return _.extend( {}, state, {
+				amount: { dataEntered: true, isValid: true }
+			} );
+		} else if ( action.type === 'INPUT_AMOUNT' ) {
+			return _.extend( {}, state, {
+				amount: { dataEntered: true, isValid: null }
+			} );
+		}
+		if ( action.type === 'CHANGE_CONTENT' && action.payload.contentName === 'paymentType' ) {
```

Comment:
You can also make this an `else if` I think.

Reply:
oopsie